packages/wonder-blocks-popover/components/popover.js

```diff
@@ -121,12 +117,6 @@ type State = {|
      * Current popper placement
      */
     placement: Placement,
-    /**
-     * Wether the Popover is controlled or uncontrolled.
-     * This helps us to know if we need to set an initial focus inside the
-     * popover dialog or not.
-     */
-    controlled: boolean,
```

Comment:
It's related to one of my previous comments:

> So when the Popover is opened on page load what is the behavior and how does this differ from it being opened some time after the page is loaded?

My answer: That's a really good point, I'm thinking that maybe it would be good to steal focus no matter if the Popover is controller or uncontrolled. At the end, we are displaying the popover, so we should inform to the user to focus on this element.

That said, do you think is still a good idea to keep the `controlled` prop for these cases? If that's the case I can revert my changes related to that.

Reply:
This does feel like a question for the design team.  TBH, I'm good dropping this.  I think whether or not to steal focus is something that should maybe be decided on a case by case basis to start with and if a pattern emerges we can build it into the library at that point.